daprdocs/content/en/operations/components/pluggable-components/pluggable-components-overview.md

```diff
@@ -0,0 +1,122 @@
+| ---------------------------- | :--------------------------------------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Language**                 | Can only be written in Go                                                                                  | [Can be written in any gRPC-supported language](https://grpc.io/docs/what-is-grpc/introduction/#:~:text=Protocol%20buffer%20versions,-While%20protocol%20buffers&text=Proto3%20is%20currently%20available%20in,with%20more%20languages%20in%20development) |
+| **Where it runs**            | As part of the Dapr executable itself                                                                      | As distinct process, container or pod. Runs seperate from Dapr itself                                                                                                                                                                                      |
+| **Integration with Dapr**    | Integrated directly into Dapr codebase                                                                     | Integrates with Dapr via Unix Domain Sockets (using gRPC )                                                                                                                                                                                                 |
+| **Hosting**                  | Hosted in Dapr repository                                                                                  | Hosted in your own repository                                                                                                                                                                                                                              |
+| **Distribution**             | Distributed with Dapr release (i.e., new features added to component need to be aligned with Dapr releases | Distributed independently from Dapr itself (i.e., new features can be added _whenever_ and follow your release cycle)                                                                                                                                      |
+| **How component is started** | Dapr starts component (automatic)                                                                          | User starts component (manual)                                                                                                                                                                                                                             |
+
+## When to create a pluggable component
+
+- You have homegrown components you want to use with Dapr APIs that can't be contributed to the project.
```

Comment:
```suggestion
- You have private components you want to use with Dapr APIs that can't be contributed to the project.
```

Reply:
Same. I was going to ask about this use of private. There is nothing in how pluggable components work that require them to be "private". I understand that we prefer folks contributing built-in ones back to the project/community but this push for private sounds off.